frontend/src/job/ingest-journal/JobParameters.ts

```diff
@@ -0,0 +1,69 @@
+
+class JournalMetadata {
```

Comment:
Wären die Klassen vielleicht besser [interfaces](https://www.typescriptlang.org/docs/handbook/interfaces.html)?

Reply:
In dem Zusammenhang auch interessant für die weitere Entwicklung: https://stackoverflow.com/questions/13320568/can-i-create-a-typescript-type-and-use-that-when-ajax-returns-json-data/13320802#13320802